examples/flexvolume/README.md

```diff
@@ -53,22 +53,60 @@ Driver will be invoked with 'Init' to initialize the driver. It will be invoked
 ### Driver invocation model:
 
 Init:
-\<driver executable\> init
+
+```
+<driver executable> init
+```
 
 Attach:
-\<driver executable\> attach \<json options\>
+
+```
+<driver executable> attach <json options>
+```
 
 Detach:
-\<driver executable\> detach \<mount device\>
+
+```
+<driver executable> detach <mount device>
+```
 
 Mount:
-\<driver executable\> mount \<target mount dir\> \<mount device\> \<json options\>
+
+```
+<driver executable> mount <target mount dir> <mount device> <json options>
+```
 
 Unmount:
-\<driver executable\> unmount \<mount dir\>
+
+```
+<driver executable> unmount <mount dir>
+```
 
 See lvm[lvm] for a quick example on how to write a simple flexvolume driver.
 
+### Driver output:
+
+Flexvolume expects the driver to reply the status of the operation in the
+following format.
+
+```
+{
+	"status": "<Success/Failure>",
+	"message": "<Reason for success/failure>",
+	"device": "<Path to the device attached>"
+}
+```
+
+### Default Json options
+
+In addition to the user specified options, the following options are passed by default to the driver.
```

Comment:
I'd suggest:

In addition to the options specified by the user in the `Options` field of the `FlexVolumeSource`, the following options are also passed to the executable.